src/GLONASS_IFB_CORRECTION/glonass_IFB_correction.c

```diff
@@ -0,0 +1,412 @@
+/*
+ *   GLONASS INTER-FREQUENCY PHASE BIASES (IFB) CORRECTION MODULE
+ */
+
+#include "glonass_IFB_correction.h"
+
+#define MIN(x,y)    ( (x) <= (y) ? (x) : (y) )
+
+/* -------------------------------------------------------------------------------------------------------------- */
+/* API */
+
+extern void *glo_IFB_init()
```

Comment:
"glonass IFB: modify API to use generic pointers" is not an explanation to use void pointers. These should be avoided at all costs. I don't see a reason to use them. Please, either describe in detail why do you need this or refactor. This looks very nasty.